docs/examples/image_quality/structural_similarity_index.jl

```diff
@@ -0,0 +1,82 @@
+# ---
+# cover: assets/ssim.png
+# title: Structural Similarity Index
+# ---
+
+# When comparing images, the **Mean Squared Error** (or MSE), though straightforward to calculate,
+# may not be a very good indicator of their *perceived* similarity.
+
+# The **Structural Similarity Index** (or SSIM) aims to address this shortcoming by taking texture
+# into account, and assigning a higher score to images that may *appear* similar.
+
+
+using Images, TestImages
+using Random
+
+img_orig = Gray{Float64}.(testimage("cameraman"))
```

Comment:
```suggestion
img_orig = float64.(testimage("cameraman"))
```

We have `float`/`float64`/`float32` that preserves the colorant information. Thus `float64(rgb_img)` works as expected, too.